app/hydrator/services/create/stores/config-store.js

```diff
@@ -111,6 +111,13 @@ class HydratorPlusPlusConfigStore {
         this.setNumRecordsPreview(this.state.config.numOfRecordsPreview);
         this.setMaxConcurrentRuns(this.state.config.maxConcurrentRuns);
       }
+      this.setPushdownConfig({
+        // null: getDefaultConfig, undefined: imported json, {}: just in case
+        enabled: (this.state.config.transformationPushdown !== null
+          && this.state.config.transformationPushdown !== undefined
```

Comment:
i think in this case, doing `!this.state.config.transformationPushdown` would suffice.

btw, why would imported JSON have `{}`?

Reply:
Hmm, I don't remember why I was thinking that. I think it may have been caused by something I was doing while testing, before I'd figured out the dataflow into config-store.js. Simplification reasonable, will remove.

Reply:
Looks like in javascript, `!null === true`, so I can only remove the object check.

Reply:
I just realized I might be an idiot. You meant `!!`, didn't you?

Reply:
sorry yeah, i meant `!!`